http-server-jetty/src/test/groovy/io/micronaut/servlet/jetty/JettyStaticResourceResolutionSpec.groovy

```diff
@@ -146,7 +149,6 @@ class JettyStaticResourceResolutionSpec extends Specification implements TestPro
         file.exists()
         response.code() == HttpStatus.OK.code
         response.header(CONTENT_TYPE) == "text/html"
-        Integer.parseInt(response.header(CONTENT_LENGTH)) > 0
```

Comment:
please keep these in, if necessary assert that `response.header(CONTENT_LENGTH) == null` instead. i am working on changes here so i want to be aware of behavior differences between versions